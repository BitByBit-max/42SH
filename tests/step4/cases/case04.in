fruit="kiwi"
case $fruit in "apple") echo "apple is tasty";;
"banana") echo "I like banana";;
"kiwi") echo "Newzeland is famous for kiwi";;
*)
Echo "default case";;
esac
