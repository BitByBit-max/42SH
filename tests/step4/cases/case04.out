Newzeland is famous for kiwi
